### PR TITLE
Load spinners

### PR DESCRIPTION
**Pull request by llmlks**

### CODE REVIEW

File: views/dataview.py [hunk 1]
```diff
@@ -7,4 +9,4 @@ def get_layout(session_id):
     Keywords arguments:
     session_id -- ID of the current session
     """
-    return data_table.get_component(session_id)
+    return dcc.Loading(data_table.get_component(session_id))
```
Comment (afhuertass) [inline views/dataview.py]: Perhaps it is better if the function `data_table.get_component` returns the loading component wrapper, does it make a difference?

**Review by afhuertass — APPROVED** [review]
I don't see any issues over here, this is good to go :+1:

File: views/dataview.py (hunk 1 above)
Comment (llmlks) [reply views/dataview.py]: Good idea, I've now changed it

**Comment by afhuertass** [comment]
Approved for maximum merge.

**Pull request merged**